packages/browser-tests/package.json

```diff
@@ -4,16 +4,31 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "start": "run-s start:*",
-    "start:setup": "node ./src/setup-example.js",
-    "start:build": "node ./src/build-example.js",
-    "start:serve": "npx serve -p 8080 --no-port-switching ./example",
-    "test": "npx playwright test"
+    "start": "npm run start:server",
+    "start:serve": "cd ../headless-tests && npx serve -p 8080 --no-port-switching . -s",
+    "test": "npx playwright test",
+    "build:headless": "cd ../headless-tests && npm run build",
```

Comment:
I think we should be able instead of `cd ../headless-tests && npm run build` to do something like `npm run build --workspace=headless-tests` in this and similar places, removing `cd ../headless-tests` altogether

let me know if it works, I am not 100% sure